cms/djangoapps/contentstore/rest_api/v1/views/tests/test_course_waffle_flags.py

```diff
@@ -72,7 +86,10 @@ def expected_response(self, enabled=False):
         Returns:
             dict: A dictionary with each flag set to the value of `enabled`.
         """
-        return {flag: enabled for flag in self.course_waffle_flags}
+        res = {flag: enabled for flag in self.course_waffle_flags}
+        for flag in self.other_expected_waffle_flags:
+            res[flag] = enabled
+        return res
```

Comment:
nit: maybe more readable to combine the 2 lists first

Reply:
That's actually much more hassle then it seems like because of the string concatenation and the places the list is used. I tried to do that and it's just taking up too much time, so in the end I decided just for a test it doesn't seem worth it. Since you also made this a "nit" I would skip it